fairlearn/widget/js/src/fairlearnDashboard.tsx

```diff
@@ -60,6 +60,7 @@ export class FairlearnView extends DOMWidgetView {
             predictedY={data.predicted_ys}
             trueY={data.true_y}
             modelNames={data.model_names}
+            customMetrics={[]}
```

Comment:
I believe we need to chat about this at some point. However it's not MVP for the AzureML integration